lightning/src/ln/channel.rs

```diff
@@ -1657,6 +1658,30 @@ pub(super) struct FundingScope {
 	funding_transaction: Option<Transaction>,
 }
 
+#[cfg(not(any(test, fuzzing)))]
+impl_writeable_tlv_based!(FundingScope, {
+	(0, value_to_self_msat, required),
+	(1, counterparty_selected_channel_reserve_satoshis, option),
+	(2, holder_selected_channel_reserve_satoshis, required),
+	(3, holder_max_commitment_tx_output, required),
+	(4, counterparty_max_commitment_tx_output, required),
+	(5, channel_transaction_parameters, (required: ReadableArgs, 0)), // FIXME: This won't work
```

Comment:
Not quite sure how we want to fix this. The `ReadableArgs` implementation checks that the read value is equal to arg value.